File: src/main/java/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/dto/SellerOrderItemsStatusUpdateRequest.java
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.dto;

import com.multicampus.gamesungcoding.a11ymarketserver.feature.order.entity.OrderItemStatus;
import jakarta.validation.constraints.NotNull;

public record SellerOrderItemsStatusUpdateRequest(
        @NotNull OrderItemStatus status) {
}

File: src/main/kotlin/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/dto/SellerOrderItemsStatusUpdateRequest.kt
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.dto

import com.multicampus.gamesungcoding.a11ymarketserver.feature.order.entity.OrderItemStatus

data class SellerOrderItemsStatusUpdateRequest(
    val status: OrderItemStatus
)
